### PR TITLE
use new person database structure

### PR DESCRIPTION
**Pull request by wow-such-code**
* fetches correct fields from new person table when fetching users

was easier than expected...

### CODE REVIEW

**Comment by wow-such-code** [comment]
[Update Project wizard to deal with the new user database structure](https://trello.com/c/IAhzzEno/755-update-project-wizard-to-deal-with-the-new-user-database-structure)

**Review by jenniferboedker — APPROVED** [review]
🚀  LGTM!

File: CHANGELOG.rst [hunk 1]
```diff
@@ -9,10 +9,14 @@ This project adheres to `Semantic Versioning <https://semver.org/>`_.
...
 **Added**
 
+* Uses new database structure for persons and affiliations
+
 **Fixed**
 
 **Dependencies**
 
+* New database structure using "person" instead of "persons" table
+
```
Comment (jenniferboedker) [inline CHANGELOG.rst:19]: I would argue that this is not a dependency change per se 🤔

File: CHANGELOG.rst [hunk 2]
```diff
@@ -9,10 +9,14 @@ This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 **Added**
 
+* Uses new database structure for persons and affiliations
+
```
Comment (jenniferboedker) [inline CHANGELOG.rst:13]: maybe state here which table you now use

**Review by jenniferboedker — APPROVED** [review]

**Pull request merged**